internal/handlers/tigris/msg_getlog.go

```diff
@@ -35,48 +38,84 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, lazyerrors.Error(err)
 	}
 
-	if l := document.Map()["getLog"]; l != "startupWarnings" {
-		errMsg := fmt.Sprintf("MsgGetLog: unhandled getLog value %q", l)
-		return nil, common.NewErrorMsg(common.ErrNotImplemented, errMsg)
-	}
+	command := document.Command()
 
-	info, err := h.driver.Info(ctx)
+	getLog, err := document.Get(command)
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
-	var log types.Array
-	for _, line := range []string{
-		"Powered by FerretDB " + version.Get().Version + " and Tigris " + info.ServerVersion + ".",
-		"Please star us on GitHub: https://github.com/FerretDB/FerretDB and https://github.com/tigrisdata/tigris",
-	} {
-		b, err := json.Marshal(map[string]any{
-			"msg":  line,
-			"tags": []string{"startupWarnings"},
-			"s":    "I",
-			"c":    "STORAGE",
-			"id":   42000,
-			"ctx":  "initandlisten",
-			"t": map[string]string{
-				"$date": time.Now().UTC().Format("2006-01-02T15:04:05.999Z07:00"),
-			},
-		})
+	if _, ok := getLog.(string); !ok {
+		return nil, common.NewErrorMsg(common.ErrTypeMismatch, "Argument to getLog must be of type String")
+	}
+
+	var resDoc *types.Document
+	switch getLog {
+	case "*":
+		resDoc = must.NotFail(types.NewDocument(
+			"names", must.NotFail(types.NewArray("global", "startupWarnings")),
+			"ok", float64(1),
+		))
+
+	case "global":
+		log, err := pg.RequirRecordsLog(zapcore.DebugLevel)
```

Comment:
Tigris handler should not import pg handler, they should be independent. I think that function could be moved to util/logging.